sdks/python/apache_beam/transforms/core.py

```diff
@@ -518,12 +525,12 @@ def _process_argspec_fn(self):
   def is_process_bounded(self):
     """Checks if an object is a bound method on an instance."""
     if not isinstance(self.process, types.MethodType):
-      return False # Not a method
+      return False  # Not a method
```

Comment:
Format of some lines are changed after I reformat the code.
Code changes start from line 2289.

Reply:
How did you reformat the code? As long as linter still passes, this is fine. However it generally easier to review when format changes are separated from other changes.

Reply:
I used reformat tool provided by IntelliJ. However, I don't think this tool is the most optimal one and I changed the format back.